src/ui/components/Passport/Passport.tsx

```diff
@@ -30,8 +32,8 @@ type PrimaryPanelName = "events" | "cypress" | string;
 const stepNames = ["step-one", "step-two", "step-three", "step-four"] as const;
 
 const Passport = (props: PropsFromRedux) => {
-  const [showTestsuitesPassportWelcome, setshowTestsuitesPassportWelcome] = useState(
-    localStorage.getItem("TestsuitesPassportWelcome") !== "true"
+  const [showTestsuitesPassportFirstRun, setShowTestsuitesPassportFirstRun] = useState(
```

Comment:
:hand: I think the `useState` here can be entirely replaced by the `useGraphQLUserData` hook from `packages/shared/user-data/`.

Reply:
I'm sure you're right, but when I tried it the banner didn't show up because the logic was:

* Show the banner!
* Ok, now that we did that, userData.set should set this to false!
* ... meaning the banner doesn't appear, even on the first render

Whereas state lets it show on the first load before disappearing. Investigating now.

Reply:
Hmm. Okay, yeah, I think the fact that you are wanting to clear it out right away is the distinction here, and maybe the `useState` is necessary.